lib/LaTeXML/Core/Document.pm

```diff
@@ -416,7 +416,9 @@ sub finalize_rec {
       # Remove any pending declarations that can't be on $FONT_ELEMENT_NAME
       my $elementname = $pending_declaration{element}{value} || $FONT_ELEMENT_NAME;
```

Comment:
Could you check if adding immediately after this line
```
      delete $pending_declaration{element};    # If any...
```
solves the problem (instead of the code below)?  Your version makes even more obscure the intention of the "element" NON-attribute.

Reply:
Certainly that works too -- just tested on my passing sTeX branch, all looks good. I rebased on master as well for extra safety. Good to merge?